lib/realtime_web/channels/realtime_channel.ex

```diff
@@ -240,6 +259,13 @@ defmodule RealtimeWeb.RealtimeChannel do
     {:noreply, socket}
   end
 
+  @impl true
+  def handle_info(%Broadcast{event: "presence_diff", payload: payload}, socket) do
```

Comment:
intercepting `presence_diff` messages here. for some reason the `handle_out` wasn't triggering but this was.

Reply:
Nice. handle_out re-serializes everything anyways.